docs/source/technical_tutorials/working_with_r/access_aws_maap.ipynb

```diff
@@ -0,0 +1,1577 @@
+{
```

Comment:
<p>What does <code>stac_endpoint[[2]]</code> get us? You may want to split this out and add a comment, or at least a meaningful var name to help the reader understand what you're pulling out of <code>stac_endpoint</code>. Perhaps something along these lines:</p><p> </p>

<pre class="ql-syntax" spellcheck="false"># Element 2 of stac_endpoint is blah, blah, blah, ...
blah_blah_blah = stac_endpoint[[2]]
stac_query &lt;- blah_blah_blah |&gt;
  stac() |&gt;
  stac_search(collections = collection) |&gt;
  get_request()  
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/471/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>